src/pybind/mgr/dashboard/controllers/rbd.py

```diff
@@ -15,7 +16,7 @@
 from .. import mgr
 from ..security import Scope
 from ..services.ceph_service import CephService
-from ..tools import ViewCache
+from ..tools import ViewCache, str_to_bool
```

Comment:
Use ``from distutils.util import strtobool`` if possible.

Reply:
if `strtobool` does everything `str_to_bool` does, then we should replace it everywhere.

Reply:
I will have a look